### DISCUSSION TITLE
Tfjs-node VS Tfjs-node-gpu performance difference ?

### DISCUSSION THREAD

**Discussion by uzair004** [vladmandic/human#162] [Q&A] [answered]
Apparently tfjs-node-gpu are intended to be used on GPU Backend having Nvidia with CUDA technology.

#### Cost vs performance of using tfjs-node-gpu on digital ocean ?

**Comment by vladmandic** [comment]
to make the answer more generic, i'll write a benchmarking tool, ETA tuesday

**Reply by uzair004** [reply]
Ok Thanks

**Comment by vladmandic** [comment, marked as answer]
actually, i just added notes under <https://github.com/vladmandic/human/wiki/Backends>
